applet/src/console.c

```diff
@@ -70,7 +64,7 @@ static void con_addchar( char c )
 {
     // intentional non-wrap.
     if( con_xpos >= MAX_XPOS ) {
-        return ;
+        con_nl();
```

Comment:
easy yes. clean no. 
can't you create a pullreq for all the other stuff you have, so we can implenent the syslog scrolling later?